components/Footer/LongFooter.tsx

```diff
@@ -71,10 +71,6 @@ export default function LongFooter({ locale = 'en' }: { locale?: string }): Reac
           url: 'https://docs.makerdao.com/dai.js',
           title: 'Dai.js'
         },
-        {
```

Comment:
Do we want to remove this link from the footer?

Reply:
it was a duplicate, the same link to Documentation appears at the top of the list